src/app/(proper_react)/(redesign)/(authenticated)/user/welcome/EnterInfo.tsx

```diff
@@ -403,7 +400,8 @@ export const EnterInfo = ({
             </Button>
           )}
           <Button
-            {...confirmDialogTrigger.triggerProps}
+            onPress={() => setConfirmDialogIsOpen(true)}
+            aria-haspopup="dialog"
```

Comment:
Weird that `triggerProps` doesn't set it then! Since this is somewhat surprising (i.e. the expectation would be that react-aria sets the correct `aria-*` attributes as needed), it might be a good idea to add a comment so that someone coming across this in the future knows if that exception still holds. And the docs you so helpfully linked are helpful to future readers too :)

```suggestion
            // react-aria sets `aria-expanded`, which causes this to be
            // announced as "collapsed", rather than opening a dialog. By
            // setting aria-haspopup, this interaction is clearer. See also:
            // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-haspopup
            aria-haspopup="dialog"
```

Reply:
I'm even slightly tempted to file a bug at react-aria, but I'm not confident enough about what the pattern should be :sweat_smile: